### PR TITLE
Accessible accordion

### PR DESCRIPTION
**Pull request by aliuk2012**
The Claim details Accordion will now support keyboard navigation.

Another major change in the way it works is that default all accordions will be closed except if the user is using a "Messages" link which will instead open just the messages panel for them when the page is loaded

### CODE REVIEW

**Comment by aliuk2012** [comment]
I seem to have a flicker on this PR with the rspec. I'm not sure how changing the view/js would affect the rspecs that are failing.

**Comment by himal-mandalia** [comment]
The accordion.js needs to be broken up and refactored with better management of scopes. Code Climate isn't happy.

File: app/assets/javascripts/accordion.js [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+  //
+  // @return N/A
+  //
+  togglePanel : function ($tab) {
+
+    var $panel = this.$panel.find('#' + $tab.attr('aria-controls'));
+
+    if ($panel.attr('aria-hidden') == 'true') {
+      $panel.attr('aria-hidden', 'false');
+      $tab.find('img').attr('src', 'http://www.oaa-accessibility.org/media/examples/images/expanded.gif').attr('alt', 'expanded');
```
Comment (Demwunz) [inline app/assets/javascripts/accordion.js]: @aliuk2012  I don't think we should be hotlinking images

**Comment by aliuk2012** [comment]
Well spotted @Demwunz, we dont actually need that and I forgot to take that out during the refactor.

**Comment by aliuk2012** [comment]
@Demwunz if you happy with it could you merge it?

**Pull request merged**